projects/pastanaga/src/lib/textfield/textfield.common.ts

```diff
@@ -16,22 +16,22 @@ export class TextfieldCommon implements ControlValueAccessor, OnInit, Validator
     @Input() placeholder?: string;
     @Input() help?: string;
     @Input()
-    get required(): boolean { return this._required; }
-    set required(value: boolean) { this._required = coerceBooleanProperty(value); }
+    get isRequired(): boolean { return this._required; }
+    set isRequired(value: boolean) { this._required = coerceBooleanProperty(value); }
     protected _required = false;
     @Input() pattern?: RegExp;
     @Input() min?: number;
     @Input() max?: number;
     @Input()
-    get disabled(): boolean { return this._disabled; }
-    set disabled(value: boolean) { this._disabled = coerceBooleanProperty(value); }
+    get isDisabled(): boolean { return this._disabled; }
+    set isDisabled(value: boolean) { this._disabled = coerceBooleanProperty(value); }
     protected _disabled = false;
     @Input() isReadOnly = false;
     @Input() isLabelHidden = false;
-
+    @Input() isPlaceholderShown = false;
```

Comment:
if possible, always use getters and setters with `coerceBooleanProperty` when creating new boolean input on a component. It's cleaner as it allow to use the property name on its own on the component:
```html
<pa-textarea isPlaceholderShown ...>
```
instead of having to do a binding like
```html
<pa-textarea [isPlaceholderShown]="true" ...>
```

By the way, you can replace all the other boolean inputs which are not using getter/setter to have a cleaner component at the end

Reply:
oh yes, you are right.. let me change it